sympy/solvers/tests/test_ode.py

```diff
@@ -2336,8 +2336,6 @@ def test_unexpanded_Liouville_ODE():
     sol2 = Eq(f(x), log(x/(C1 + C2*x)))
     sol2s = constant_renumber(sol2)
     assert dsolve(eq2) in (sol2, sol2s)
-    assert checkodesol(eq2, sol2, order=2, solve_for_func=False)[0]
```

Comment:
What's happened here? If checkodesol worked  for sol2 before why wouldn't it now?

Reply:
It is working fine. I have removed it by mistake.